### PR TITLE
[SYCLomatic] Fix missing migration of cudaStream_t arg for 8 CUB APIs

### PR DESCRIPTION
**Pull request by yihwang-nv**
Fix missing migration of cudaStream_t arg for 8 CUB APIs

Signed-off-by: Wang, Yihan <yihan.wang@intel.com>

### CODE REVIEW

File: clang/lib/DPCT/CallExprRewriter.cpp [hunk 1]
```diff
@@ -2596,6 +2593,16 @@ class CheckArgIsConstantIntWithValue {
   }
 };
 
+class CheckArgIsDefaultCudaStream {
```
Comment (wenhuiNi) [inline clang/lib/DPCT/CallExprRewriter.cpp:2622]: Maybe the function checkIsArgStream provides the similar function.

Comment (yihanwg) [reply clang/lib/DPCT/CallExprRewriter.cpp:2596]: I can't found `checkIsArgStream` in this repo

**Review by tomflinda — APPROVED** [review]
LGTM

**Review by AndyCHHuang — APPROVED** [review]
LGTM

**Review by wenhuiNi — APPROVED** [review]

**Pull request merged**